Examples/example_esp_async.py

```diff
@@ -0,0 +1,43 @@
+# The MIT License (MIT)
+# Copyright (c) 2020 Mike Teachman
+# https://opensource.org/licenses/MIT
+
+# example for MicroPython rotary encoder
+#
+# Documentation:
+#   https://github.com/MikeTeachman/micropython-rotary
+
+from rotary_irq_esp import RotaryIRQ
+import uasyncio as aio
```

Comment:
In MicroPython, most contributors use:

`import uasyncio as asyncio`

Reply:
yeah, just being too lazy. lol